zebra-chain/src/block/difficulty/tests.rs

```diff
@@ -233,20 +364,32 @@ fn expanded_hash_order() -> Result<(), Report> {
 }
 
 proptest! {
-   /// Check that CompactDifficulty expands without panicking, and compares
-   /// correctly.
+    /// Check that CompactDifficulty expands, and converts to work.
+    ///
+    /// Make sure the conversions don't panic, and that they compare correctly.
    #[test]
-   fn prop_compact_expand(compact in any::<CompactDifficulty>()) {
-       // TODO: round-trip test, once we have ExpandedDifficulty::to_compact()
-       let expanded = compact.to_expanded();
+   fn prop_compact_expand_work(compact in any::<CompactDifficulty>()) {
+        // TODO: use random ExpandedDifficulties, once we have ExpandedDifficulty::to_compact()
+        //
+        // This change will increase the number of valid random work values.
+     let expanded = compact.to_expanded();
+       let work = compact.to_work();
 
        let hash_zero = BlockHeaderHash([0; 32]);
        let hash_max = BlockHeaderHash([0xff; 32]);
 
+       let work_zero = Work(0);
+       let work_max = Work(u128::MAX);
+
        if let Some(expanded) = expanded {
            prop_assert!(expanded >= hash_zero);
            prop_assert!(expanded <= hash_max);
        }
+
+       if let Some(work) = work {
+            prop_assert!(work > work_zero);
```

Comment:
`fmt`ing?

Reply:
```suggestion
           prop_assert!(work > work_zero);
```

Reply:
Nice catch, my editor is meant to fix those automatically.